kubernetes/linera-validator/grafana-dashboards/linera-general.json

```
@@ -107,11 +552,12 @@
           },
           "disableTextWrap": false,
           "editorMode": "builder",
+          "exemplar": false,
           "expr": "rate(block_execution_latency_bucket{service=\"shards\"}[$__rate_interval])",
```

Doesn't need to be on this PR, but maybe we should change this to show the actual amounts, and maybe be the p99 of the last 10 minutes or something?
Something like this:

```suggestion
          "expr": "histogram_quantile(0.99, rate(block_execution_latency_bucket{service=\"shards\"}[10m]))",
```